static/shared/js/typeahead.js

```diff
@@ -153,10 +153,17 @@ export function sort_emojis(objs, query) {
 
     const triage_results = triage(query, others, (x) => x.emoji_name);
 
+    function prioritise_realm_emojis(emojis) {
+        return [
+            ...emojis.filter((emoji) => emoji.is_realm_emoji),
+            ...emojis.filter((emoji) => !emoji.is_realm_emoji),
+        ];
+    }
+
     const sorted_results_with_possible_duplicates = [
         ...popular_emoji_matches,
```

Comment:
My parsing is that means that adding a local `thumbs_up` will still render after the default "popular" one.  Which I think is a reasonable compromise?

I note this comment at the top of the file:
> The caveat is that other factors still may matter more, such as prefix matches trumping "popularity".
...which doesn't seem accurate?  A popular match is always going to be at the front of the result list.

Reply:
Right, have removed that line in another commit here. We already have a comment "For typeahead we'll favor any of these as long as the emoji code matches." so this didn't really need any replacement.